[Swift 6]: Update Expert mixolgist and its concept (#833)
exercism/swift: concepts/control-transfer/about.md

# Introduction

Swift has a set of statements that can be used to alter the normal control flow of loops.
These are known as control transfer statements.
You have already seen `return`; this concept will introduce other, which include `continue`, `break`, and labels.

> We won't cover `fallthrough` in this concept, but you can read about it in the [Swift book][fallthrough].

Reviewer: Is Fallthrough covered? I see it below.

Author: The introduction doesn't cover it to keep the text concise but you are right, that shouldn't be in the about docs.

// grapes
```

## Labels

When loops are nested, there are times when one may want to use `break` or `continue` to exit or restart the outer loops that contain the loop in which the `break` or `continue` are used.
In cases like these, labels may be used to specify the loop to be exited or restarted.
A loop can be labeled by putting a name followed by a colon before the `while`, `repeat`, or `for` that starts the loop.

To see the effect of using labels in this way, consider the following loop.

```swift
outerLoop: for fruit in ["banana", "grapes", "apple", "strawberry", "kiwi", "lemon"] {
  print("\n--- \(fruit) ---")
  for letter in fruit {
    guard letter != "n" else { break outerLoop }
    print(letter)
  }
}

// prints:
// --- banana ---
// b
// a
```

> ## Fallthrough

Reviewer: ```suggestion
## Fall through
```

Reviewer: If you mean the keyword, it should be lowercase and not titlecase.

The `fallthrough` keyword is used to fall through to the next case in a switch statement.

> Meaning that the code in the next case will be executed, even if the value in the next case does not match the value being compared.

Reviewer 2: ```suggestion
This means the the code within the next case will be executed without checking its condition(s).
```
